src/view/components/tab/Tab.tsx

```diff
@@ -0,0 +1,27 @@
+import { Pivot, PivotItem, PivotLinkFormat } from "office-ui-fabric-react";
+import * as React from "react";
+import { CONSTANTS, DEVICE_LIST_KEY } from "../../constants";
+
+interface IProps {
+    handleTabClick: (item?: PivotItem) => void;
+}
+export class Tab extends React.Component<IProps, any> {

```

Comment:
If it doesn't have a state, consider using React.FC

Also, instead of any, I'd use {} empty state to pass to React.Component